.goreleaser.yml

```diff
@@ -0,0 +1,47 @@
+project_name: fastly
+release:
+  prerelease: auto
+before:
+  hooks:
+    - go mod tidy
```

Comment:
I totally understand why we want to do this during the release (to make sizes smaller) but it seems like our test suite should run this and fail the build if there are extraneous dependencies. Thoughts?

Reply:
I like the idea of ensuring we have a clean go.mod and go.sum file via a CI test :+1: